aries-core/src/js/components/helpers/Anchors/FooterLink.js

```diff
@@ -9,7 +9,7 @@ export const FooterLink = ({ children, label, size, ...rest }) => {
     <Anchor
       color={textColor}
       label={
-        <Text size={size} weight="bold">
+        <Text size={size} weight="normal">
```

Comment:
```suggestion
        <Text size={size} >
```
I'm thinking if it's normal, then we probably won't need this prop. I might be wrong though.

Reply:
at first I thought the same but Im guessing there is somewhere in the theme that is still making it bold so when I took out normal it was still bold

Reply:
Theme has font-weight bold, so we need to specify normal here.  At least until we do the work for https://github.com/hpe-design/design-system/issues/308

Reply:
Okay just looked because `Text` is being wrapped in the `Anchor` and in the theme for `Anchor`  is  
```
 anchor: {
    color: 'text',
    fontWeight: 700,
    textDecoration: 'none',
    hover: {
      textDecoration: 'none',
    },
```
so that fontWeight 700 is being passed down to the Text therefore I need the normal